nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/versioncheck/VersionInfo.java

```diff
@@ -94,7 +94,8 @@ public VersionInfo(URI uri) throws IOException {
 
         //Extract values from properties file:
         Properties properties = new Properties();
-        try (InputStream is = uri.toURL().openStream() ) {
+        URL u = new URL(path);  //Can't use URI.toUrl() due to spaces in path
```

Comment:
@AlexDBlack can you add a constant to this representing the character, and then clean up the comments? I'd be happy with this after that.